dev/tool/README.md

```diff
@@ -0,0 +1,105 @@
+## How to Import Documents from Notion
+
+To import Notion documents:
+
+1. Export documents from Notion as *Markdown & CSV*
+2. Extract the exported archive
+3. 1. If you want your docs to be imported along with Notion teamspaces, use this command:
+
+
+```
+rushx run-local import-notion-with-teamspaces ${dir} \
+--user ${user} \
+--password ${password} \
+--workspace ${workspace}
+```
+
+3.  2. If you want the docs to be imported to a separate new teamspace, use this command:
+
+```
+rushx run-local import-notion-to-teamspace {dir} \
+--user ${user} \
+--password ${password} \
+--workspace ${workspace} \
+--teamspace ${teamspace}
+```
+
+* *dir* - path to the root of the extracted archive
+* *user* - your username or email
+* *password* - password
+* *workspace* - workspace name where the documents should be imported to
+* *teamspace* - teamspace to be created for newly imported docs
+
+
+### Examples
+
+#### For local run:
+When importing Notion workspace with teamspaces
+```
+rushx run-local import-notion-with-teamspaces /home/Export-fad9ecb4-a1a5-4623-920d-df32dd423743 \
+--user john.doe@gmail.com \  
+--password qwe1234 \       
+--workspace ws1
+```
+When importing Notion workspace without teamspaces or a page with subpages
+```
+rushx run-local import-notion-to-teamspace /home/Export-fad9ecb4-a1a5-4623-920d-df32dd423743 \
+--user john.doe@gmail.com \
+--password qwe1234 \
+--workspace ws1 \
+--teamspace imported
+```
+
+#### For clouds:
+##### Here is an example for those who's using huly.app cloud:
+To import Notion workspace with teamspaces
+```
+docker run \
+  -e SERVER_SECRET="" \
```

Comment:
May be we fix tool to not ask for secret if not used?
ACCOUNTS_URL could be retrieved from FROMT_URL/config.json so better to add this fix as well.

Reply:
done